src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/AvoidStaticImportCheckTest.java

```diff
@@ -91,17 +93,18 @@ public void testMemberExcludes()
             throws Exception {
         final DefaultConfiguration checkConfig =
             createModuleConfig(AvoidStaticImportCheck.class);
-        checkConfig.addAttribute("excludes", "java.io.File.listRoots");
+        checkConfig.addAttribute("excludes", "java.io.File.listRoots,java.lang.Math.E");
```

Comment:
`java.lang.Math.E` it is added to exclusions, but I don't see it in test input file.

Reply:
I initially put it just to add as a complement to Math.sqrt for the edge case. 
I've added it in the appropriate places now.